_benchmark/reference/summary-report.md

```diff
@@ -0,0 +1,260 @@
+
+## Translog size
+
+**Corresponding metrics key**: `translog_size_in_bytes`
+
+The translog size in bytes as reported by the Index Stats API.
+
+## Heap used for X
+
+**Corresponding metrics keys**: `segments_*_in_bytes`
+
+The number of bytes used for the corresponding item as reported by the Index Stats API. The item may be any of the following:
+
+- Doc values
+- Terms
+- Norms
+- Points
+- Stored fields
+
+
+## Segments count
+
+**Corresponding metrics key**: `segments_count`
+
+The total number of segments as reported by the Index Stats API.
+
+
+## Total ingest pipeline count
+
+**Corresponding metrics key**: `ingest_pipeline_cluster_count`
+
+The total number of documents ingested by all nodes within the cluster over the race duration.
+
+## Total ingest pipeline time
+
+**Corresponding metrics key**: `ingest_pipeline_cluster_time`
+
+The total time in milliseconds spent preprocessing ingest documents by all nodes within the cluster over the race duration.
+
+
+## Total ingest pipeline failed
+
+**Corresponding metrics key**: `ingest_pipeline_cluster_failed`
+
+The total number of failed ingest operations by all nodes within the cluster over the race duration.
+
+
+## Throughput
+
+**Corresponding metrics key**: `throughput`
+
+Reports the minimum, mean, median, and maximum throughput for each task.
+
+The number of operations that OpenSearch can perform within a certain time period  per second. The report includes the minimum, mean, median, and maximum throughput for each task.
+
+
+## Latency
+
+**Corresponding metrics key**: `latency`
+
+The time period between submission of a request and receiving the complete response. It includes the wait time the request spends waiting before it is processed by OpenSearch. OpenSearch reports several percentile numbers for each task. Which percentiles are shown depends on how many requests OpenSearch can capture during the latency period.
+
+
+## Service time
+
+**Corresponding metrics key**: `service_time`
+
+The time period between sending a request and receiving the corresponding response. It does not include waiting time. While many load testing tools refer to this metic as _latency_, it is not the same. OpenSearch reports several percentile numbers for each task. Which percentiles are shown depends on how many requests OpenSearch can capture during the latency period.
+
+
+
+## Processing time
+
+Processing time is only reported if the setting `output.processingtime` is set to `true` in the OpenSearch Benchmark configuration file.
+{: note.}
+
+**Corresponding metrics key**: `processing_time`
+
+
+The time period between start of request processing and receiving the complete response. Unlike `service_time`, this metric includes OpenSearch’s client-side processing overhead. Large differences between `service_time` and `processing_time` indicate a high overhead in the client and thus can point to a potential client-side bottleneck that requires investigation.
```

Comment:
```suggestion
The time period between start of request processing and retrieval of the complete response. Unlike `service_time`, this metric includes OpenSearch’s client-side processing overhead. Large differences between `service_time` and `processing_time` indicate a high overhead in the client and thus can point to a potential client-side bottleneck that requires investigation.
```

Reply:
What constitutes a "large difference"? A certain number of seconds? We should give the user a clear example; otherwise, it's subjective.